content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Understanding Synthetic Tests and Alerting Monitors
```

Comment:
- Any reason why we are talking about `Alerting Monitors` and not just `Monitors` here? 
- Also, it feels like this doc is not really only about "Understanding" the relationship between the two objects, right? Several sections feel more like tips on how to use monitors to answer some specific use cases, wdyt?

Reply:
"Tips and tricks to benefit from your Synthetic test monitor"? Lol might be too much